src/probnum/diffeq/odefiltsmooth/initialize.py

```diff
@@ -0,0 +1,284 @@
+"""Initialisation procedures."""
+# pylint: disable=import-outside-toplevel
+
+
+import numpy as np
+import scipy.integrate as sci
+
+import probnum.filtsmooth as pnfs
+import probnum.random_variables as pnrv
+from probnum.filtsmooth import statespace as pnss
+
+__all__ = [
```

Comment:
I know I already asked a long time ago, and this has nothing to do with the PR, but: Why exaclty is this `__all__` needed here? Those are the only two functions in this file anyways, so `from initialize import *` should already have the desired behaviour?

Reply:
it makes sure that e.g. `pnrv` is not available through importing `initialize`. (And, as a mini side result, it gives some documentation)

Reply:
As far as I know, it is available as `import initialize; initialize.pnrv`, but not when importing with `from initialize import *`?